content/cli/workspaces.md

```diff
@@ -294,9 +294,12 @@ TypeScript compilation automatically distributes compiler output (`.js` and `.d.
 The value of the `assets` key should be an array of elements specifying the files to be distributed. The elements can be simple strings with `glob`-like file specs, for example:
 
 ```typescript
-"assets": ["**/*.graphql"]
+"assets": ["**/*.graphql"],
+"watchAssets": true,
 ```
 
+> info **Hint** To watch your assets during development, use `watchAssets` option
+
 For finer control, the elements can be objects with the following keys:
 
 - `"include"`: `glob`-like file specifications for the assets to be distributed
```

Comment:
I think we don't need the Hint, but rather a warning like this (below the code sample)

I think the *warning* here is about the parent setting overriding the child setting.  So how about we change this to:

> error **Warning** setting `watchAssets` in a top-level `compilerOptions` property overrides any `watchAssets` settings within the `assets` property.

I think that's slightly confusing, but more accurate.  If it still seems confusing, we could put something like your example in, but I don't think it's really needed.

Reply:
Totally agree. I don't think we need the example, the warning message is enough. Already updated.